presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/RetryPolicyType.java

```diff
@@ -27,6 +27,7 @@
     DOWNGRADING_CONSISTENCY(DowngradingConsistencyRetryPolicy.INSTANCE),
     FALLTHROUGH(FallthroughRetryPolicy.INSTANCE);
 
+    @SuppressWarnings("ImmutableEnumChecker")
```

Comment:
Why?

Reply:
`RetryPolicy` is not a immutable to error-prone plugin (it must be annotated with `@Immutable`).

So it's violate [ImmutableEnumChecker](https://errorprone.info/bugpattern/ImmutableEnumChecker) that some of fields in enum is mutable.